reactor-netty-core/src/main/java/reactor/netty/channel/MonoSend.java

```diff
@@ -58,7 +58,8 @@ static <O> ToIntFunction<O> defaultSizeOf() {
 		return (ToIntFunction) SIZE_OF;
 	}
 
-	static final int                    MAX_SIZE    = 128;
+	static final int                    MAX_SIZE =
+			Integer.parseInt(System.getProperty("reactor.netty.channel.send.prefetch.maxSize", "128"));
```

Comment:
I think we can drop `channel`
```suggestion
			Integer.parseInt(System.getProperty("reactor.netty.send.prefetch.maxSize", "128"));
```

Reply:
@pderop What do you think about the name of the system property?

Reply:
I am first wondering if , for consistency reasons, should this new property be declared (or not ?) in the ReactorNetty class ? (most of reactor netty system properties are defined in the `reactor.netty.ReactorNetty`).

To shorten the property name, we indeed may drop `channel`.

now, based on our existing system properties, what about `reactor.netty.send.maxPrefetch` or  `reactor.netty.send.maxPrefetchSize` ? this is more consistent with our existing system properties like "reactor.netty.logChannelInfo" which are generally ending with a camel case string ?

Reply:
@pderop I agree

@VadimKirilchuk Can you move the system property to `ReactorNetty` and rename it to `reactor.netty.send.maxPrefetchSize`?

Reply:
@violetagg Sure, just busy at the moment, will do asap.
I like new property name.